### PR TITLE
Add Tech Industry Night 2022

### PR DESCRIPTION
**Pull request by kewbish**
This PR adds the event page for the 2022 Tech Industry Night.

Please let me know if there's anything you'd like me to add / change / fix!

### CODE REVIEW

File: content/events/2022-01-27-tech-industry-night.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+end_date: 2022-01-27 19:00:00
+---
+
+![Tech Industry Night 2022 Banner](/files/tech-industry-night-2022.jpg)
+
+The CSSS will host the first 2022 Tech Industry Night during the inaugural CS week which runs from January 24th-31st.
+
+Students will have the opportunity to interact with industry professionals and recruiters from tech companies and learn more about the job market, interview strategies, resume tips, and more!
+
+Sign up by scanning the QR code on the poster!
```
Comment (eqfy) [inline content/events/2022-01-27-tech-industry-night.md:25]: Can you also add this signup survey link?
https://ubc.ca1.qualtrics.com/jfe/form/SV_3a8WDxJVILuRiku

**Review by eqfy — APPROVED** [review]
Thanks for adding the event!

I am actually wondering if by merging it in, does netlify build the website using the the preview or the code on the merged master? I guess we can find out

**Comment by kewbish** [comment]
@eqfy no problem! Added the survey link - could you just double check that's what you wanted?

**Comment by eqfy** [comment]
@kewbish Everything looks good!

**Pull request merged**